app/gui/view/component-browser/component-list-panel/grid/src/entry/icon.rs

```diff
@@ -44,16 +44,48 @@ define_icons! {
         }
     }
 
+    /// A compass icon. Eight-pointed star.
+    /// NOTE: This icon is not used in the Component Browser, but rather in the visualization
+    /// container. It is placed here temporarily, to demonstrate it in the icons demo scene easily.
+    /// It will be moved when refactoring the visualization container.
```

Comment:
Added

Reply:
In fact, we should once move it to a separate crate, gathering "all icons for GUI view". 
1. Those icons may be used both as shapes as well as read from cached shapes (and icons in visualization chooser may be taken from cache, as they are not zoomed in).
2. The Libraries may want to use this icon for some entries. Because why not?

And the node's action bar icons should be merged in as well. I'll create a task for doing it.

Reply:
We didn't do that in the past because of layering issues in ensogl. (it was impossible to render icons from a separate crate correctly) 

As far as I remember, it should be fixed now.